src/commands/upgrade.rs

```diff
@@ -51,7 +52,6 @@ impl UpgradeCommand {
             //
             // Skip this in tests, but still delegate to a new instance of `cargo-component`
```

Comment:
Here's the rationale for the original approach. If this is changing, this bit of the comment should probably go, too.

Reply:
Oh whoops, yeah, I'll remove the comment. Thanks!